common/common.js

```diff
@@ -37,6 +37,20 @@ function restrictReferences(utils, content) {
   return (base.innerHTML);
 }
 
+// Mark definitions to be exported
+function exportReferences(utils, content) {
+  const base = document.createElement("div:not([data-cite])");
+  base.innerHTML = content;
+
+  const defns = base.querySelectorAll("dfn");
+  for (const item of defns) {
```

Comment:
I suggest not  doing this... ReSpec  automatically exports the matching WebIDL definitions for you, so you don't need to worry about them. 

For the  rest, exports should  be very deliberate: only export things that are actually referenced by  other specs.

Reply:
Then, I need something for the other specs to not export the from common files, or does this happen because the first spec to get into the database prevents the others from being externalized?

Reply:
So, ideally, the other specs won’t need to include the common definitions anymore as the definitions will be linked to the main spec (unless you really want them, in which case the imported duplicates need to be marked with “data-no-export” to avoid redefinitions).

Reply:
We do want the definitions in the other specs, so I’ll need to add data-cite to the other spec versions to reference them from the API doc which defines them, I suppose.

Reply:
ok, that's fine. My personal preference is not to do that (i.e., just link, don't duplicate things across specs - like other web platform specs that build on each other) - but if duplicating definitions across specs is  convention in the JSON-LD community, that's ok. 

The core thing is that we have only one source of truth for the definitions, and to make sure we only export what is really needed by other specs.

Reply:
Re-reading, your first comment, ReSpec automatically exports WebIDL definitions, but this function is used in for local term bibliographies for terms shared between the different specs, so I think it should remain so that they get exported.

The other specs will use `data-cite` to reference, so the sections will look the same, but they won't actually make the definitions and references will redirect to the API document.

I thought this was the point of adding `xref` to the config, so that the namespace for definitions was made common, and I wouldn't need to do these things manually, but I think I don't really understand the point of xref and why some definitions would go into your database and others wouldn't.

In my mind, the `xref` config would give a set of common specifications so that definitions created in any would use one of them (the first, I suppose) as the base for the definition and ReSpec would take care of anything else. I don't really care that definitions go into a database, just that URLs that are generated are unique.